views/post_tags.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+import json
+from .views_helper import dict_factory
+
+database = "./db.sqlite3"
+
+
+def get_all_post_tags():
+    with sqlite3.connect(database) as conn:
+        conn.row_factory = dict_factory
+        db_cursor = conn.cursor()
+
+        db_cursor.execute(
+            """
+            SELECT
+                pt.id,
+                pt.post_id,
+                pt.tag_id,
+                t.label
+                FROM PostTags pt
+                JOIN Tags t
+                    ON t.id = pt.tag_id
+             """
+        )
+        query_results = db_cursor.fetchall()
+
+        post_tags = []
+        for row in query_results:
+            post_tag = {
+                "post_tag_id": row["id"],
```

Comment:
May want to change this to be just `"id"`.

Reply:
I have pushed the updated version that (I think) fixes those error issues and adjusts the PostTags id to just say id